packages/kit/src/runtime/server/page/load_data.js

```diff
@@ -312,6 +312,36 @@ export function create_universal_fetch(event, state, fetched, csr, resolve_opts)
 					});
 				}
 
+				if (key === 'body') {
+					const body = response.body;
+					if (!body) return body;
+					const [a, b] = body.tee();
```

Comment:
any concerns about memory? response data will be copied to two locations in memory if I understand this right.